test/unit/archivesspaceSpec.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+describe('ArchivesSpace', function() {
+  beforeEach(module('archivesSpaceService'));
+  beforeEach(angular.mock.inject(function(_$httpBackend_) {
+    _$httpBackend_.when('GET', '/archivesspace.json').respond([
+      {
+        'dates': '2015-01-01',
+        'title': 'Test fonds',
+        'levelOfDescription': 'Test fonds',
+        'children': [],
+        'sortPosition': 2,
+        'identifier': 'F1',
+        'id': '/repositories/2/resources/1',
+      },
+    ]);
+  }));
+
+  it('should be able to return a list of all ArchivesSpace records', inject(function(_$httpBackend_, ArchivesSpace) {
```

Comment:
In the relatively short term I think we're going to be fetching only parent records via `all()` and children via a new method, so I'll wait to add a test for that until later.